cmd/skupper/skupper.go

```diff
@@ -61,6 +61,7 @@ type SkupperServiceClient interface {
 	UnbindFlags(cmd *cobra.Command)
 	Unexpose(cmd *cobra.Command, args []string) error
 	UnexposeFlags(cmd *cobra.Command) error
+	UnExposeArgs(cmd *cobra.Command, args []string) error
```

Comment:
I would keep it as `UnexposeArgs` with lowercase `e` (just to follow the existing standard).

Reply:
Updated in e9cd7496aa1548fcb4cbb73325afa2ab6347badd